themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -1551,3 +1551,9 @@ tls-client-certificate-bound-access-tokens=OAuth 2.0 Mutual TLS Certificate Boun
 tls-client-certificate-bound-access-tokens.tooltip=This enables support for OAuth 2.0 Mutual TLS Certificate Bound Access Tokens, which means that keycloak bind an access token and a refresh token with a X.509 certificate of a token requesting client exchanged in mutual TLS between keycloak's Token Endpoint and this client. These tokens can be treated as Holder-of-Key tokens instead of bearer tokens.
 subjectdn=Subject DN
 subjectdn-tooltip=A regular expression for validating Subject DN in the Client Certificate. Use "(.*?)(?:$)" to match all kind of expressions.
+
+# KEYCLOAK-10747 Explicit Proof Key for Code Exchange Activation Settings
+pkce-activation=Proof Key for Code Exchange Activation
+pkce-activation.tooltip=If On, keycloak refuses client's authorization request without PKCE's code challenge.
+pkce-code-challenge-method=Proof Key for Code Exchange Code Challenge Method
```

Comment:
I suggest to not have 2 properties, but have only single property "Requested PKCE Method" . It can be empty, plain or S256. In case that empty value is used (which is by default), the PKCE method is not enforced and would be just read from the "code_challenge_method" parameter as it is now. It will be good to add this information to the tooltip as well. This will allow us to have one less configuration parameter, which is always better :)

Reply:
Yes, you are right. I'll implement only one configuration parameter as you said.